### PR TITLE
Tabs: Fixed the missed ids and wrong tabIndex issues

### PR DESCRIPTION
**Pull request by DancuVladTrimble**
modus-tabs component

## Description

Last week we integrated the modus-tabs component in our UI application and noticed two small issues:

- the tabs generated by the modus-tabs component do not have an id in HTML even if there is an id property in the tab configuration. This problem affects our QA to access tabs in e2e tests (in our tests we access the HTML element by id);
- all tabs have the same tabIndex = 0;

References: #2117

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

I ran all the e2e and unit tests from your application and my changes didn t affect any tests(all tests passed).

I tested locally and everything looks good, now each tab has the right id set in the tab configuration, also each tab has an unique tabIndex(0, 1, 2, 3,....) :

image

before my changes:

image

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-tabs/modus-tabs.tsx [hunk 1]
```diff
@@ -57,12 +57,12 @@ export class ModusTabs {
     const tabs = this.tabs.map((tab: Tab) => {
       return (
         <div
+          id={`${tab.id}`}
           class={`tab ${tab.active ? 'active' : ''} ${this.classBySize.get(this.size)} ${
             this.fullWidth ? 'resizable' : ''
           } `}
           onClick={() => this.handleTabChange(tab.id)}
-          onKeyDown={(event) => this.handleKeyDown(event, tab.id)}
-          tabIndex={0}>
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-tabs/modus-tabs.tsx:65]: Without a tabIndex, the tab won't be able to be tabbed into.

File: stencil-workspace/src/components/modus-tabs/modus-tabs.tsx [hunk 2]
```diff
@@ -57,12 +57,12 @@ export class ModusTabs {
     const tabs = this.tabs.map((tab: Tab) => {
       return (
         <div
+          id={`${tab.id}`}
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-tabs/modus-tabs.tsx:60]: I need to double check how we want to handle this.  I need to familiarize myself with the component first, but we have utility functions for generating IDs and I need to check if we desire to have the ID given by the user to map directly to the element or not. I'll follow-up on this point soon.

File: stencil-workspace/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trimble-oss/modus-web-components",
-  "version": "0.15.0",
+  "version": "0.16.0",
```
Comment (cjwinsor) [inline stencil-workspace/package.json:3]: You don't need to update the version, we will take care of this when we release.

Comment (DancuVladTrimble) [reply stencil-workspace/package.json:3]: Ok, got it

File: stencil-workspace/src/components/modus-tabs/modus-tabs.tsx (hunk 2 above)
Comment (DancuVladTrimble) [reply stencil-workspace/src/components/modus-tabs/modus-tabs.tsx:60]: we need to set an id for each tab base on the tabs configuration:

image

without an id our QA is not able to access from e2e tests these tabs.

File: stencil-workspace/src/components/modus-tabs/modus-tabs.tsx (hunk 1 above)
Comment (DancuVladTrimble) [reply stencil-workspace/src/components/modus-tabs/modus-tabs.tsx:65]: @coliff asked me to delete the tabIndex. Also I tested without this property and everything is working like before.

**Review by cjwinsor — APPROVED** [review]

**Pull request merged**